daemon/src/projection.rs

```diff
@@ -481,6 +500,37 @@ impl Cfd {
             }
         }
     }
+
+    fn derive_actions(&self, role: Role) -> HashSet<CfdAction> {
+        match (self.state, role) {
+            (CfdState::PendingSetup, Role::Maker) => {
+                HashSet::from([CfdAction::AcceptOrder, CfdAction::RejectOrder])
+            }
+            (CfdState::PendingSetup, Role::Taker) => HashSet::new(),
+            (CfdState::ContractSetup, _) => HashSet::new(),
+            (CfdState::Rejected, _) => HashSet::new(),
+            (CfdState::PendingOpen, _) => HashSet::new(),
+            (CfdState::Open, _) => HashSet::from([CfdAction::Commit, CfdAction::Settle]),
+            (CfdState::PendingCommit, _) => HashSet::new(),
+            (CfdState::PendingCet, _) => HashSet::new(),
+            (CfdState::PendingClose, _) => HashSet::new(),
+            (CfdState::OpenCommitted, _) => HashSet::new(),
```

Comment:
🤔 I think the behaviour has changed here when coming from `CetTimelockConfirmedPriorOracleAttestation`: we used to keep the available actions from applying the previous event.

But this looks correct to me since there's no action to do when `commit_tx` is on-chain.